perfmetrics/scripts/fetch_metrics.py

```diff
@@ -0,0 +1,33 @@
+"""Executes fio_metrics.py and vm_metrics.py by passing appropriate arguments.
+"""
+import socket
+import sys
+import time
+from fio import fio_metrics
+from vm_metrics import vm_metrics
+
+START_TIME = 'start_time'
+END_TIME = 'end_time'
+INSTANCE = socket.gethostname()
+PERIOD = 120
+
+if __name__ == '__main__':
+  argv = sys.argv
+  if len(argv) != 2:
+    raise TypeError('Incorrect number of arguments.\n'
+                    'Usage: '
+                    'python3 fetch_metrics.py <fio output json filepath>')
+
+  fio_metrics_obj = fio_metrics.FioMetrics()
+  print('Getting fio metrics...')
+  temp = fio_metrics_obj.get_metrics(argv[1])
+  print('Waiting for 250 seconds for metrics to be updated on VM...')
```

Comment:
can we add a new line here to seperate out fio metrics logic and vm metrics logic